### PR TITLE
Fixing menu leftside

### PR DESCRIPTION
**Pull request by eerison**

### CODE REVIEW

**Comment by eerison** [comment]
Screenshot 2022-03-02 at 22 29 12
 
 
the layout of left side is quite ok, but I'm struggle to make this dropdown works, do you have any idea what is the problem @jordisala1991

**Comment by jordisala1991** [comment]
Hi! Can you avoid all the indentation changes? That should go to another PR. Let's try to keep the original PR as clean as possible, with the minimum changes required to make this work.

File: src/Resources/views/standard_layout.html.twig [hunk 1]
```diff
@@ -203,8 +203,8 @@ file that was distributed with this source code.
                                 {% if sonata_config.getOption('search') %}
                                     <form action="{{ path('sonata_admin_search') }}" method="GET" class="sidebar-form" role="search">
                                         <div class="input-group custom-search-form">
-                                            <input type="text" name="q" value="{{ app.request.get('q') }}" class="form-control" placeholder="{{ 'search_placeholder'|trans({}, 'SonataAdminBundle') }}">
-                                            <span class="input-group-btn">
+                                        <input type="text" name="q" value="{{ app.request.get('q') }}" class="form-control form-control-sidebar" placeholder="{{ 'search_placeholder'|trans({}, 'SonataAdminBundle') }}">
```
Comment (jordisala1991) [inline src/Resources/views/standard_layout.html.twig]: This indentations should be fixed

File: src/Resources/views/standard_layout.html.twig [hunk 2]
```diff
@@ -236,10 +236,10 @@ file that was distributed with this source code.
...
                             {% block sonata_page_content_nav %}
                                 {% if _navbar_title is not empty
-                                  or _tab_menu is not empty
-                                  or _actions is not empty
-                                  or _list_filters_actions is not empty
-                                 %}
+                                    or _tab_menu is not empty
+                                    or _actions is not empty
+                                    or _list_filters_actions is not empty
+                                %}
```
Comment (jordisala1991) [inline src/Resources/views/standard_layout.html.twig]: same indentation needs change here

**Comment by eerison** [comment]
I guess I removed the indentation changes 😄 

@jordisala1991 do you know how to add this js/css files : https://github.com/ColorlibHQ/AdminLTE/issues/4304#issuecomment-1059935816

if not I can check how to add this :) 

Edit 1: do you know if this line

https://github.com/sonata-project/SonataAdminBundle/pull/7740/files#diff-2126216ecd2ff9ae9b5f0c1e84d8fecb4ee4e54dc21685f69a590c2162dec186L41

import: scrollbarOverlays and adminlte (cs/js) ?

**Review by jordisala1991 — APPROVED** [review]

**Comment by jordisala1991** [comment]
Thank you!

**Comment by jordisala1991** [comment]
> I guess I removed the indentation changes 😄 
> 
> 
> 
> @jordisala1991 do you know how to add this js/css files : https://github.com/ColorlibHQ/AdminLTE/issues/4304#issuecomment-1059935816
> 
> 
> 
> if not I can check how to add this :) 
> 
> 
> 
> Edit 1: do you know if this line
> 
> 
> 
> https://github.com/sonata-project/SonataAdminBundle/pull/7740/files#diff-2126216ecd2ff9ae9b5f0c1e84d8fecb4ee4e54dc21685f69a590c2162dec186L41
> 
> 
> 
> import: scrollbarOverlays and adminlte (cs/js) ?

Do you know if scrollbarOverlays is part of admin lte or is it another library?

To add it, we should know that first, but it should be added to the same file we included the main admin lte js, searching for it on node_modules. 

I can do it if you want. Just dont have a lot of time for OSS.

**Comment by jordisala1991** [comment]
I have done some research, maybe is this one: https://github.com/KingSora/OverlayScrollbars

It might need to be installed and added to css/js

**Comment by eerison** [comment]
I think it make part of adminlte, I didn't see this in extra components 👀👀

Well if you could add this will be nice, because i Will unblock me, and I
could see others things related with adminlte V3.

**Comment by eerison** [comment]
Hmmmm does it should be added by yarn and webpack? And after that add in
app.js?

**Comment by jordisala1991** [comment]
> Hmmmm does it should be added by yarn and webpack? And after that add in
> app.js?

Thats the idea, but if you want to take a look on the other parts of the compatibility I can take care of this

**Comment by eerison** [comment]
Nice then I'll fix the list view ;)

**Pull request merged**